gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/experimental/dsl/IosDeployConfigurations.kt

```diff
@@ -18,12 +19,32 @@ open class IosDeployConfigurations @Inject constructor(
         configureSimulator.execute(currentSimulator)
         deployTargets.add(DeployTargetWithId(id, currentSimulator))
     }
+
+    public fun localFile(id: String, configureLocalFile: Action<DeployTarget.LocalFile>) {
+        val current = objects.newInstance(DeployTarget.LocalFile::class.java)
+        configureLocalFile.execute(current)
+        deployTargets.add(DeployTargetWithId(id, current))
+    }
+
+    public fun connectedDevice(id: String, configureConnectedDevice: Action<DeployTarget.ConnectedDevice>) {
+        val current = objects.newInstance(DeployTarget.ConnectedDevice::class.java)
+        configureConnectedDevice.execute(current)
+        deployTargets.add(DeployTargetWithId(id, current))
+    }
+
 }
 
 sealed interface DeployTarget {
     open class Simulator : DeployTarget {
         var device: IOSDevices = IOSDevices.IPHONE_8
```

Comment:
Have you considered pulling this out to a separate plugin, as it seems useful in general for any KMP iOS project?

Reply:
Good idea! We are considering this option.